teams/migration/membership.json

```diff
@@ -45,19 +43,22 @@
         "overvenus",
         "sdojjy",
         "suzaku",
+        "Tammyxia",
+        "WangXiangUSTC",
+        "WizardXiao",
+        "YuJuncen",
+        "zhangjinpeng87",
         "zhaoxinyu",
-        "zwj-coder",
-        "hongyunyan",
```

Comment:
sorry in old list they are sorted in ASCII order, where the uppercase is always smaller than lowercase 😂 I only mean to move "hongyunyan" and "lidezhu" to correct place